src/solvers/dgmulti/dg.jl

```diff
@@ -243,10 +243,25 @@ function max_dt(u, t, mesh::DGMultiMesh,
   # `polydeg+1`. This is because `nnodes(dg)` returns the total number of
   # multi-dimensional nodes for DGMulti solver types, while `nnodes(dg)` returns
   # the number of 1D nodes for `DGSEM` solvers.
-  polydeg = rd.N
   return 2 * dt_min / (polydeg + 1)
 end
 
+function max_dt(u, t, mesh::DGMultiMesh,
+                constant_speed, equations, dg::DGMulti{NDIMS}, cache) where {NDIMS}
+  rd = dg.basis
+  if dg.basis.approximation_type isa TensorProductWedge
+    # In the computation of max_dt the polynomial degree is used to compute the time-step.
+    # For tensor-product elements the maximum of the polynomial degrees is used to ensure
+    # that the timestep is feasable for both elements of the tensor-product. For the
+    # other elements taking the maximum has no effect.
+    polydeg=maximum(rd.N)
+  else
+    polydeg = rd.N
+  end
+  return max_dt(u, t, mesh, constant_speed, equations, dg, polydeg, cache)
+end
```

Comment:
```suggestion
function max_dt(u, t, mesh::DGMultiMesh,
                constant_speed, equations, dg::DGMulti{3, Wedge, <:TensorProductWedge}, cache) 
  rd = dg.basis
  # In the computation of max_dt the polynomial degree is used to compute the time-step.
  # For tensor-product elements the maximum of the polynomial degrees is used to ensure
  # that the timestep is feasable for both elements of the tensor-product. For the
  # other elements taking the maximum has no effect.
  polydeg=maximum(rd.N)
  return max_dt(u, t, mesh, constant_speed, equations, dg, polydeg, cache)
end
```
What do you think of this? To me, this is clearer that the routine is specialized for tensor product wedges.